src/styles/main.scss

```diff
@@ -0,0 +1,12 @@
+@import './variables';
+
+body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  display: flex;
```

Comment:
The `display: flex;` property should be placed before `flex-direction`, `justify-content`, and `align-items` properties to improve readability and follow the logical sequence.